host-provision.yml

```diff
@@ -38,10 +38,10 @@
   hosts: all
   vars_prompt:
     - name: ansible_password
-      prompt: Enter customeradmin password
+      prompt: Enter firsttime_connect_user password
```

Comment:
Is there a way to dereference the variable so that a user will see "Enter customeradmin password"

Reply:
There may be, but first glance, extrapolating this variable is introducing complexity that we can do away.